WcaOnRails/app/models/regulation.rb

```diff
@@ -0,0 +1,16 @@
+class Regulation < SimpleDelegator
+  begin
+    REGULATIONS = JSON.parse(File.read("app/views/search_results/wca-regulations.json"))
```

Comment:
1. I don't know, I'm up for any suggestion! We could show up some error in the search results, log this error?

2. Indeed the server needs to be restarted. How bad is it to restart the production server? There is like 2 Regulations updates a year, so it's not like with translations where we could have more frequent updates.
If it's something we really can't afford, then I need to somehow check if the file has changed and reload it if needed.